lib/common/cmn_files.ml

```diff
@@ -0,0 +1,14 @@
+open Core.Std
+open Core_extended.Std
+open Async.Std
+
+exception Cmn_write_error
+
+let write root name contents =
```

Comment:
I have reviewed that one. It has a minor comment, but is either ready for merging if you drop the comment or almost ready changing one line